diff --git a/modules/panopoly/panopoly_wysiwyg/panopoly_wysiwyg.make b/modules/panopoly/panopoly_wysiwyg/panopoly_wysiwyg.make
index 9a70f76f1..59f9b2330 100644
--- a/modules/panopoly/panopoly_wysiwyg/panopoly_wysiwyg.make
+++ b/modules/panopoly/panopoly_wysiwyg/panopoly_wysiwyg.make
@@ -28,9 +28,9 @@ projects[caption_filter][subdir] = contrib
 ; Include our Editors
 
 libraries[tinymce][download][type] = get
-libraries[tinymce][download][url] = http://download.moxiecode.com/tinymce/tinymce_3.5.11.zip
-libraries[tinymce][patch][1561882] = http://drupal.org/files/1561882-cirkuit-theme-tinymce-3.5.8.patch
-libraries[tinymce][patch][2876031] = https://www.drupal.org/files/issues/tinymce-chrome58-fix-2876031-5.patch
+;libraries[tinymce][download][url] = http://download.moxiecode.com/tinymce/tinymce_3.5.11.zip
+;libraries[tinymce][patch][1561882] = http://drupal.org/files/1561882-cirkuit-theme-tinymce-3.5.8.patch
+;libraries[tinymce][patch][2876031] = https://www.drupal.org/files/issues/tinymce-chrome58-fix-2876031-5.patch
 
 libraries[markitup][download][type] = git
 libraries[markitup][download][url] = https://github.com/markitup/1.x.git
